### PR TITLE
#2538 Update attachment size limit in rich text mode

### LINKED ISSUE #2538: forbid sending rich text messages with more than 19 mb atts
- it's hard on the recipient, especially on mobile
- gmail may not accept it due to size problems (over 20mb or so) due to several layers of encoding (size bloat)

### PR DESCRIPTION
**Pull request by martgil**
This PR updates the attachment size limit in Rich Text Editor mode.

close #2538 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by martgil** [comment]
ready for review. thank you!

File: extension/chrome/elements/compose-modules/compose-attachments-module.ts [hunk 1]
```diff
@@ -32,13 +32,16 @@ export class ComposeAttachmentsModule extends ViewModule<ComposeView> {
   };
 
   private getMaxAttachmentSizeAndOversizeNotice = async (): Promise<AttachmentLimits> => {
-    const sizeMb = 25;
+    // To prevent size bloating, it is recommended to set the attachment size limit to 19MB for Rich-text editor mode - https://github.com/FlowCrypt/flowcrypt-browser/issues/2538#issuecomment-1639926581
+    const sizeMb = this.view.sendBtnModule.popover.choices.richtext ? 19 : 25;
```
Comment (sosnovsky) [inline extension/chrome/elements/compose-modules/compose-attachments-module.ts:36]: let's use `this.view.inputModule.isRichText()` instead `this.view.sendBtnModule.popover.choices.richtext`, so it'll be consistent with other code

Comment (martgil) [reply extension/chrome/elements/compose-modules/compose-attachments-module.ts]: Thank you - I haven't spotted the predefined function too early 😅, I will use it now.

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**